src/styles/common/textStyles.js

```diff
@@ -34,6 +35,14 @@ const rawStyles = {
     fontFamily: THEME.FONTS.DEFAULT,
     fontSize: THEME.rem(1),
     textAlign: 'center'
+  },
+
+  linkText: {
```

Comment:
These styles are meant to be more primitive, like colors and fonts vs. layout like  width & alignment. Please remove width & alignment from this style.

Reply:
More to the point, they are supposed to be composable, so someone could do `style={[textStyles.bodyCenter, textStyles.linkText]}` to get the effect you are going for. So really, `linkText` should just be the color override.